### ISSUE TITLE
Core introduction method

### ISSUE THREAD

**Issue by relu91** [vaimee/zion#20]
As specified in https://w3c.github.io/wot-discovery/#introduction-core-rd-sec, the Thing Description of a Thing Description Directory may be advertised using the CoRE Link format and the CoRE Resource Directory. In [Zion](https://github.com/vaimee/zion/) we can have two modes:
- [ ] Zion registers itself into another CoRE Resource Directory
- [x] Zion host also a CoRE Resource Directory endpoint and automatically register its TD.

**Comment by relu91** [comment]
Re-opening cause #29 implemented one of the two features

**Comment by JKRhb** [comment]
>     * [ ]  Zion registers itself into another CoRE Resource Directory

The process to do this is described in [RFC 9176, section 5](https://datatracker.ietf.org/doc/html/rfc9176#section-5). For the implementation we could either let the TDD discover local CoRE-RDs (e.g., via mDNS/DNS-DS or CoAP multicast) where is then going to register or define a registration URL via the config.

However, I assume this feature is not the most important one to add at the moment.

**Comment by relu91** [comment]
> The process to do this is described in [RFC 9176, section 5](https://datatracker.ietf.org/doc/html/rfc9176#section-5). For the implementation we could either let the TDD discover local CoRE-RDs (e.g., via mDNS/DNS-DS or CoAP multicast) where is then going to register or define a registration URL via the config.

> However, I assume this feature is not the most important one to add at the moment.

Yes, exactly we can start from the configuration and later improve with automatic discovery. As you can see we labeled the issue with low priority but if there is anyone interested we can change it.
